packages/react-component-library/src/components/NumberInput/EndAdornmentButton.tsx

```diff
@@ -1,16 +1,90 @@
 import React from 'react'
 import classNames from 'classnames'
 import capitalize from 'lodash/capitalize'
+import { selectors } from '@royalnavy/design-tokens'
+import styled, { css } from 'styled-components'
 
 import { END_ADORNMENT_TYPE } from './constants'
 
+const { color } = selectors
+
+type EndAdornmentType =
+  | typeof END_ADORNMENT_TYPE.DECREASE
+  | typeof END_ADORNMENT_TYPE.INCREASE
+
 interface EndAdornmentButtonProps {
   isCondensed: boolean
   isDisabled: boolean
   onClick: (event: React.MouseEvent<HTMLButtonElement>) => void
-  type: typeof END_ADORNMENT_TYPE.DECREASE | typeof END_ADORNMENT_TYPE.INCREASE
+  type: EndAdornmentType
+}
+
+interface StyledButtonProps {
+  isCondensed: boolean
 }
 
+const StyledButton = styled.button<StyledButtonProps>`
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
+  width: 42px;
+  align-items: center;
+
+  background: transparent;
+  margin: 0;
+  padding: 0;
+  outline: 0;
+  border: 0;
+
+  flex-grow: 1;
+
+  &:focus {
+    border-radius: 4px;
+    box-shadow: 2px 2px 0 0 ${color('action', '600')},
+      -2px -2px 0 0 ${color('action', '600')},
+      2px -2px 0 0 ${color('action', '600')},
+      -2px 2px 0 0 ${color('action', '600')};
+  }
+`
+
+const StyledDecreaseButton = styled(StyledButton)`
+  border-top: 1px solid ${color('neutral', '100')};
+
+  &:focus {
+    border-color: transparent;
+  }
+
+  ${({ isCondensed }) =>
+    isCondensed &&
+    css`
```

Comment:
Hmm i've been doing this without the `css`. Is this optional?

Reply:
It seems like it is optional but my IDE does not like it and the documentation I see always seems to use `css`.

With `css`:
![Screenshot 2020-10-19 at 13 56 08](https://user-images.githubusercontent.com/56078793/96453990-263ad300-1213-11eb-9c60-a1f4dea9a59e.png)

Without `css`:
![Screenshot 2020-10-19 at 13 56 28](https://user-images.githubusercontent.com/56078793/96453971-1f13c500-1213-11eb-961d-51b79326f759.png)